app/code/Magento/ConfigurableProductGraphQl/etc/schema.graphqls

```diff
@@ -64,6 +64,7 @@ input ConfigurableProductCartItemInput {
 type ConfigurableCartItem implements CartItemInterface {
     customizable_options: [SelectedCustomizableOption] @resolver(class: "Magento\\QuoteGraphQl\\Model\\Resolver\\CustomizableOptions")
     configurable_options: [SelectedConfigurableOption!]! @resolver(class: "Magento\\ConfigurableProductGraphQl\\Model\\Resolver\\ConfigurableCartItemOptions")
+    configured_variant: ProductInterface @doc(description: "Product details of the cart item") @resolver(class: "\\Magento\\ConfigurableProductGraphQl\\Model\\Resolver\\ProductResolver")
```

Comment:
Hey @sasha19957099 , any reasons for deviating from the schema here ?

https://github.com/magento/architecture/blob/master/design-documents/graph-ql/coverage/cart/Cart.graphqls

Reply:
Hey @cpartica. I have 2 questions on this
1.  Do we need to expose store config to expose this configuration ```checkout/cart/configurable_product_image either parent or itself```. 
2. configured_variant should return ProductInterface or SimpleProduct! as per the schema?

Reply:
Resolution: in `configured_variant: ProductInterface` ProductInterface should be mandatory "!"